packages/@o3r/rules-engine/builders/rules-engine-extractor/index.ts

```diff
@@ -11,22 +11,23 @@ import {
   createBuilder,
 } from '@angular-devkit/architect';
 import {
-  CmsMetadataData,
+  type CmsMetadataData,
   createBuilderWithMetricsIfInstalled,
   getLibraryCmsMetadata,
 } from '@o3r/extractors';
-import globby from 'globby';
 import {
   MetadataFact,
   MetadataOperator,
   ObjectMetadataFact,
   RulesEngineExtractor,
 } from './helpers';
 import {
-  RulesEngineExtractorBuilderSchema,
+  type RulesEngineExtractorBuilderSchema,
 } from './schema';
+// eslint-disable-next-line @typescript-eslint/no-require-imports -- default import not working for some reason
```

Comment:
is it because of a lack of the `esModuleInteropt` config on tsconfig or can it be replaced by `import * as globby from 'globby';` ?

Reply:
I've tried adding it on the `tsconfig.json` but it didn't change anything and it's already there in `tsconfig.builders.json`

Reply:
I believe it only works because this executor is not used in the monorepo itself, contrary to the ones I had to modify

Reply:
I see.

Then maybe you can add this precision in the comment, such as:

```suggestion
// eslint-disable-next-line @typescript-eslint/no-require-imports -- default import not working for some reason when used inside Otter mono-repository
```